e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { test, expect, Page } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { Common } from '../utils/Common';
+import {
+  BackstageShowcase,
+  CatalogImport,
+} from '../support/pages/CatalogImport';
+
+test.describe.skip('GitHub Happy path', () => {
```

Comment:
So you skip nearly all migrated tests ?

Reply:
@davidfestal  It will be migrated in a separate PR. Jira tickets have been created for each test marked with skip.